docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '3'
+services:
+  database:
+    image: mtirsel/mysql-5.1
+    environment:
+      - MYSQL_DATABASE=feedback_test
+      - MYSQL_ROOT_PASSWORD=mas
+      - MYSQL_USER=mas
+      - MYSQL_PASSWORD=mas
+    volumes:
+      - database:/var/lib/mysql/data
+    restart: always
+  rails:
+    build:
+      context: .
+      dockerfile: jenkins/Dockerfile
+    environment:
+      - GITHUB_USER=$GITHUB_TOKEN_USR
+      - GITHUB_PASS=$GITHUB_TOKEN_PSW
+      - DANGER_GITHUB_API_TOKEN=$GITHUB_TOKEN_PSW
+      - CHANGE_ID=$DANGER_CHANGE_ID
+      - GIT_URL=$DANGER_GIT_URL
+      - JENKINS_URL=$DANGER_JENKINS_URL
+      - HTTP_REQUEST_TIMEOUT=10
+      - LANG=C.UTF-8
+    depends_on:
+      - database
+    links:
+      - database
+volumes:
+  bundle: {}
```

Comment:
Any idea what 'bundle' is for? I have a suspicion it's not needed.

Reply:
No idea at all, I just copied the template applied by @osharpe in multiple projects.